src/views/v2/month/event-multiday.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * View: Month Event Multiday
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/event-multiday.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+
+$event    = $this->get( 'event' );
+$event_id = $event->ID;
+
+$classes = [ 'tribe-events-calendar-month__event-multiday' ];
+
+if ( $is_featured = tribe( 'tec.featured_events' )->is_featured( $event_id ) ) {
+	$classes[] = 'tribe-events-calendar-month__event-multiday--featured';
+}
+?>
+<div class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>">
+	<?php if ( $is_featured ) : ?>
```

Comment:
Same comment I made to Paul around our icon usage, especially for cases like this where there is importance to what this icon represents. Let's be sure to either add a11y hidden text noting this is a featured event or add a `title` & `aria-label` to communicate that.

Last thing, I'd recommend going with an `<em>` tag for this particular implementation as it will give it a bit more semantic juice and importance.

cc @paulmskim, good idea to do where you have implemented as well ; )

Reply:
@ryanurban Yup, haven't forgotten ;)